### DISCUSSION TITLE
Remove All Selected Items When Changing The Page

### DISCUSSION THREAD

**Discussion by NigelDorning** [livewire/livewire#3900] [1. Help]
Hi,

I've been through the screencasts video on bulk selection and have a select all checkbox in my table. However I want to remove all those selected items when the page is updated. 

I've tried using an updatedPage lifecycle hook but for some reason the $page property is updated when I tick the select all checkbox to update the $selectPage property. As far as I'm aware the $page property shouldn't be updating then, only when the user clicks to the next page.

Thanks

**Comment by joshhanley** [comment]
@NigelDorning there was an issue with the `updatedPage` lifecycle hook triggering on all requests that was patched in 2.6.5. So please upgrade to the latest version 2.6.7 and let us know if the issue is still happening.

Hope this helps!
